tests/test_segment.py

```diff
@@ -67,6 +67,11 @@ def __unit_test_structure_function(metric):
         # And that the metric is 0
         assert np.allclose(score, 0)
 
+    # Test for non-matching numbers of intervals and labels
```

Comment:
Yeah... :laughing: I guess coveralls is too smart to allow me to do that since I didn't change the associated source file. It should improve overall coverage though.